src/test/java/org/openrewrite/github/AddCronTriggerTest.java

```diff
@@ -176,4 +177,9 @@ void cronTriggerRandom(String cronExpression, String actualCronValue) {
           )
         );
     }
+
+    @Test
+    void constructRecipeWithRecipeIntrospectionUtilsShouldNotFail() {
+        RecipeIntrospectionUtils.constructRecipe(AddCronTrigger.class);
+    }
```

Comment:
Maybe remove this test again following https://github.com/openrewrite/rewrite/pull/3420
```suggestion
```

Reply:
yes, let's wait until we merge the extra check on rewriteRun, and then we will be able to get rid of all those "extra weird tests"